### ISSUE TITLE
Error reindex

### ISSUE THREAD

**Issue by ruggeroland** [ivancukns/magento2-ddev#9]
Hi,
I have installed the mile/elasticsuite plugin and when I launch bin/magento indexer:reindex I have this error

ElasticSuite Category Indexing index process error during indexation process:
{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"Unknown filter type [phonetic] for [phonetic]"}],"type":"illegal_argument_exception","reason":"Unknown filter type [phonetic] for [phonetic]"},"status":400}

The Smile ElasticSuite for Magento 2 requires 2 plugins: analysis-phonetic & analysis-icu
This link details what is required: https://github.com/Smile-SA/elasticsuite/wiki/ServerConfig-5.x#installing-required-plugins
It states "Plugins can be installed with the bin/elasticsearch-plugin tool of Elastic Search.":

bin/elasticsearch-plugin install analysis-phonetic
bin/elasticsearch-plugin install analysis-icu

is it possible to add these to your opensearch image?

Best regards

**Comment by ivancukns** [comment]
@ruggeroland Hey there,

thanks for bringing this to my attention!

I didn't realize myself, that Magento supports Opensearch version 2+ from now on.
So I upgraded the version also here.

Have in mind that we are using here the official Opensearch Docker image https://hub.docker.com/r/opensearchproject/opensearch/ version 2.5.0 and not my own image.

I made a custom Dockerfile and added automatic installations of the opensearch plugins.

I tested it with Elasticsuite - it should work now! 

I threw in the required config in the env.php file, so it should work out of the box.

Checkout the version 2.0.1 of this repo to get the new stuff.

Cheers
